patches/appstream/0001-Add-Managarm-support.patch

```diff
@@ -1,42 +1,59 @@
-From 896f9880f471a051a2ac474c542965a6dc906701 Mon Sep 17 00:00:00 2001
+From 8b19332567d488d6303077035602a7e7326cfd2f Mon Sep 17 00:00:00 2001
 From: Dennis Bonke <admin@dennisbonke.com>
-Date: Sun, 21 Jan 2024 15:45:52 +0100
+Date: Tue, 17 Dec 2024 21:14:37 +0100
 Subject: [PATCH] Add Managarm support
 
 Signed-off-by: Dennis Bonke <admin@dennisbonke.com>
 ---
+ data/meson.build     | 4 ++--
+ meson.build          | 4 +++-
  src/as-system-info.c | 3 +++
- src/meson.build      | 3 +--
- 2 files changed, 4 insertions(+), 2 deletions(-)
+ 3 files changed, 8 insertions(+), 3 deletions(-)
 
+diff --git a/data/meson.build b/data/meson.build
+index 7ae7a9b..c724ef3 100644
+--- a/data/meson.build
++++ b/data/meson.build
+@@ -9,8 +9,8 @@ install_data('its/metainfo.loc',
+ 
+ # Do not rely on an exe wrapper for rel-info, use the system one in that case
+ if meson.is_cross_build()
+-    dependency('appstream', version: '>=' + as_version, native: true,
+-               not_found_message: 'Native appstream required for cross-building')
++#    dependency('appstream', version: '>=' + as_version, native: true,
++#               not_found_message: 'Native appstream required for cross-building')
```

Comment:
Why is this needed even if we have `host-appstream`?

Reply:
Even with `host-appstream` meson refused to find the binary with `dependency`. As the program is in path, the `find_program` call works fine, finds host-appstream and the build will succeed after that.